### ISSUE TITLE
models_WGAN.py has old keras batchnorm code

### ISSUE THREAD

**Issue by jcpeterson** [tdeboissiere/DeepLearningImplementations#55]
"mode" is no longer an acceptable keyword argument

In DeepLearningImplementations/WassersteinGAN/src/model/models_WGAN.py

**Comment by tdeboissiere** [comment]
Fixed it, thanks.
